WORKSPACE

```diff
@@ -95,3 +95,38 @@ http_archive(
     strip_prefix = "buildtools-main",
     url = "https://github.com/bazelbuild/buildtools/archive/main.zip",
 )
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
```

Comment:
Is there a reason to load this again? It's already loaded at line 2, but I'm not sure what best practices are for WORKSPACE files which appear not to prefer (with the preference enforced by buildifier) that loads happen at the start.

Reply:
Fixed.